dm/worker/worker.go

```diff
@@ -72,21 +71,14 @@ func NewWorker(cfg *Config) (*Worker, error) {
 		subTasks:    make(map[string]*SubTask),
 	}
 
-	// initial relay purger
-	operators := []purger.RelayOperator{
-		w.relayHolder,
-		streamer.GetReaderHub(),
-	}
-	interceptors := []purger.PurgeInterceptor{
-		w,
-	}
-	w.relayPurger = purger.NewPurger(cfg.Purge, cfg.RelayDir, operators, interceptors)
-
 	// initial relay holder
-	err := w.relayHolder.Init()
+	purger, err := w.relayHolder.Init([]purger.PurgeInterceptor{
```

Comment:
if `err != nil`, should we need to close `purger`, right?

Reply:
I change the logic, but I really don't want handle some resource release problem which
 in startup logic of binary process, because they will definitely be released